pages/common/adb-reboot.md

```diff
@@ -0,0 +1,20 @@
+# adb reboot
+
+> Android Debug Bridge Install: reboot an Android emulator instance or connected Android devices.
```

Comment:
This should be as brief as possible
```suggestion
> Reboot a connected Android device or emulator.
```

Reply:
TBH this was taken from `pages/common/adb-shell.md`.

but sure.

Reply:
I see, thanks. I'll make that one briefer too.